README.md

```diff
@@ -43,11 +43,13 @@ Enter the virtual environment as described above, then:
 ## How to post data.
 To post data, use /send endpoint with POST method. Data must be in the following format.
 
- {
-    "type": "TEST_MAGIC_LINK",
-    "to": "test_recipient@email.com",
-    "content": "MAGIC LINK GOES HERE"
-}
+   {
+    "type": "MAGIC_LINK",
+    "to": "ramandeep.sharma@communities.gov.uk",
+    "content": {"magic_link_url":"https://www.google.co.uk/",
```

Comment:
i'd hit a new line after the opening of the curley and before the end of the block

Reply:
Thank you. Updated 😄